packages/intl/command.js

```diff
@@ -4,6 +4,44 @@ const path = require('path');
 const { red, green, yellow, blue } = require('chalk');
 const glob = require('glob');
 
+const formatters = {
+  json: (msgs) => {
+    const results = {};
+    // removes debug info like start, file, etc
+    for (const { id, description, defaultMessage } of Object.values(msgs)) {
+      results[id] = {
+        id,
+        description,
+        defaultMessage,
+      };
+    }
+    return results;
+  },
+
+  smartling: (msgs) => {
+    const results = {
+      smartling: {
+        translate_paths: [
+          {
+            path: '*/message',
+            key: '{*}/message',
+            instruction: '*/description',
+          },
+        ],
+        variants_enabled: true,
+        string_format: 'icu',
+      },
+    };
+    for (const [id, msg] of Object.entries(msgs)) {
```

Comment:
personally I find this style a little awkward vs `Object.entries(...).map(...).fromEntries()`, but its all the same in the end